src/routes/patterns/components/Buttons.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { Button, ButtonGroup, Card } from 'components/baseline';
 
 const Buttons = () => (
-  <Card>
+  <Card className='Buttons'>
```

Comment:
yeah, as part of the build process, we have it setup to run on `git commit` at work.